src/components/ExtendedLink.tsx

```diff
@@ -0,0 +1,19 @@
+import { Link, LinkProps } from 'react-router-dom'
+
+interface Props extends LinkProps {
+  disabled?: boolean
+}
+
+export function ExtendedLink({ disabled, ...props }: Props) {
```

Comment:
I've extended the `Link` component since it doesn't have a disabled state internally